src/command/cmd_funcs.c

```diff
@@ -7533,14 +7540,17 @@ cmd_pgp(ProfWin* window, const char* const command, gchar** args)
         }
 
         ProfAccount* account = accounts_get_account(session_get_account_name());
-        char* err_str = NULL;
+        auto_char char* err_str = NULL;
```

Comment:
This declaration could/should go one block down, before its actual usage ... really minor ...